abnf/odata-abnf-construction-rules.txt

```diff
@@ -560,11 +560,14 @@ complexColPathExpr = collectionPathExpr
  
 collectionPathExpr = count [ OPEN expandCountOption *( SEMI expandCountOption ) CLOSE ]
                    / filterExpr [ collectionPathExpr ]
+                   / OPEN collectionOption *( SEMI collectionOption ) CLOSE [ complexPathExpr ] 

```

Comment:
Since the `index` is not mandatory, the `complexPathExpr` may come after a collection with more than one entry. Is this OK?

Reply:
No, we have to call that out in the prose specification.